NangoHQ/nango
# feat(logs): share an operation via URL

## Describe your changes

Fixes NAN-1013 
Fixes NAN-1003
Contributes to NAN-908

- Allow to share an operation details via URL
We just append a query param and when it's there we open the Drawer back (datadog style)

- Update copybutton with appropriate new icon

## How to test?

- Open the UI
- Open an operation
- Copy the URL into a new tab
- Drawer should open automatically
